.github/workflows/ci.yml

```diff
@@ -21,6 +19,7 @@ jobs:
       matrix:
         # Nextflow versions: check pipeline minimum
         nxf_ver: ['20.11.0-edge']
+        engine: ['docker']
```

Comment:
any reason to not add singularity and conda? To keep the tests short? I am just thinking that given we pull from 3 different sources now, this could catch incorrect urls and such quickly

Reply:
random spaces accidentally added somewhere in between has caused me a few issues here and there 😄

Reply:
Plan is to do that, but not implemented yet